examples/camera/app.js

```diff
@@ -103,7 +103,7 @@ export default function App() {
         }}
         controller={true}
         effects={[vignetteEffect, aaEffect]}
-        {...adapter.getProps({deckRef, setReady, onNextFrame, getLayers})}
```

Comment:
We get to remove React Ref from hubble's interface, which helps #22 eventually happen.